lib/shopify_cli/sed.rb

```diff
@@ -0,0 +1,19 @@
+module ShopifyCLI
+  class Sed
+    class SedError < StandardError; end
+
+    def replace_inline(filename, pattern, output)
+      command =
+        case CLI::Kit::System.os
+        when :mac
+          "sed -i ''"
+        when :linux
+          "sed -i"
+        else
+          raise "Unrecognized system!"
+        end
```

Comment:
I think we can simplify this to just support Mac, as only shopifolks will release new versions.